--- tazpanel-1.5.3/installer.cgi
+++ tazpanel-1.5.5/installer.cgi
@@ -7,7 +7,7 @@
 # Authors : Dominique Corbex <domcox@slitaz.org>
 #
 
-VERSION=0.29
+VERSION=0.30
 
 # Common functions from libtazpanel
 . lib/libtazpanel
@@ -52,7 +52,7 @@
 		# Home Partition
 		if [ -n "$(GET HOME_SELECT)" ] ; then
 			TGT_HOME=$(echo "$(GET TGT_HOME)" | sed 's/\//\\\//'g)
-			[ -n "$(GET HOME_FS)" ] && TGT_HOME_FS=$(GET HOME_FS)
+			[ -n "$(GET HOME_FMT)" ] && TGT_HOME_FS=$(GET HOME_FS) || TGT_HOME_FS=""
 		else
 			TGT_HOME=""
 			TGT_HOME_FS=""
@@ -314,7 +314,7 @@
 <a class="button" onclick="document.forms['ConfigForm'].url.value = '$(tazinst showurl stable)'; return true;">$(gettext "Stable")</a>
 <a class="button" onclick="document.forms['ConfigForm'].url.value = '$(tazinst showurl cooking)'; return true;">$(gettext "Cooking")</a>
 $(gettext "URL:")
-<input id="url" type="url" size="55" name="SRC_WEB" value="$get_SRC_WEB" placeholder="$(gettext "Full url to an ISO image file")" />
+<input id="url" type="url" size="55" name="SRC_WEB" $([ "$INST_TYPE" == "web" ] && echo -e "value=\"$SRC_FILE\"") placeholder="$(gettext "Full url to an ISO image file")" />
 </label>
 </div>
 EOT
@@ -542,7 +542,7 @@
 	local code 
 	code=0
 	# Check tazinst
-	if ! tazinst usage | grep -q Usage: ; then
+	if ! [ -x /usr/sbin/tazinst ] ; then
 		cat <<EOT
 <h3>$(gettext "Tazinst Error")</h3>
 <p><strong>tazinst</strong>, $(gettext "the lightweight SliTaz HDD installer
